packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1584,13 +1584,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.dataStores.setAttachState(attachState);
     }
 
-    public createSummary(): ISummaryTree {
+    public createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree {
```

Comment:
I'm a bit confused RE usage of blobRedirectTable.
We should store it in snapshot, right?
If not (want to understand why), shouldn't it be a boolean flag, not a map?